OrbitQt/servicedeploymanager.h

```diff
@@ -49,8 +49,8 @@ class ServiceDeployManager : public QObject {
  private:
   EventLoop loop_;
 
-  DeploymentConfiguration deployment_configuration_;
```

Comment:
since this is storing a reference we should probably make it clearer that this cannot be temporary by changing c-tor and this field to const *

Reply:
Agree. So it is not clear, that the lifetime of the configuration must be at least the same as the lifetime of the manager.